tests/TestCase/Form/FormTest.php

```diff
@@ -150,6 +150,18 @@ public function testValidateDeprected()
      * @return void
      */
     public function testErrors()
+    {
+        $this->deprecated(function () {
+            $this->testGetErrors();
```

Comment:
you no need to wrap this method with `deprecated` when it does not use/have deprecated methods